pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -75,5 +75,10 @@ func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
 		errorList = append(errorList, field.Required(field.NewPath("bootstrapInPlace", "installationDisk"),
 			"installationDisk must be set the target disk drive for the installation"))
 	}
+	if installConfig.BootstrapInPlace != nil && installConfig.Networking.NetworkType == "OpenShiftSDN" {
+		errorList = append(errorList, field.Invalid(field.NewPath("networking", "networkType"), installConfig.Networking.NetworkType,
+			"bootstrap in place does not support a NetworkType of OpenShiftSDN. Please use OVNKubernetes instead."))
```

Comment:
Bootstrap in place is kinda of impdemetation details. Lets just write that sno does not support sdn. It will align with the docs

Reply:
The other error msgs in this validation method all use "bootstrap in place". If SNO is synonymous with bootstrap in place then maybe this an opportunity to reword all the other error msgs too?